docs/streaming-programming-guide.md

```diff
@@ -641,17 +641,17 @@ methods for creating DStreams from files and Akka actors as input sources.
 
     <div class="codetabs">
     <div data-lang="scala" markdown="1">
-        streamingContext.fileStream[KeyClass, ValueClass, InputFormatClass](dataDirectory)
+        streamingContext.fileStream[KeyClass, ValueClass, InputFormatClass](dataDirectory, depth)
```

Comment:
I think all of the examples should continue to show this without the `depth` parameter, since it's optional and for particular use cases.